test/testapps/SimpleMvcFSharp/Views/Home/Index.cshtml

```diff
@@ -0,0 +1,106 @@
+﻿@{
```

Comment:
Could you simplify this to have one or two small views? We don't need the full gamut of an F# project checked in for our tests